src/printer/CMakeLists.txt

```diff
@@ -3,4 +3,4 @@
 # =============================================================================
 add_library(printer OBJECT code_printer.cpp json_printer.cpp nmodl_printer.cpp)
 set_property(TARGET printer PROPERTY POSITION_INDEPENDENT_CODE ON)
-target_link_libraries(printer PRIVATE util)
+target_link_libraries(printer PRIVATE util Backward::Interface)
```

Comment:
I don't remember precisely: does this link (or assume) the presence of libraries like libunwind? (Thinking of portability of wheels and also portability on windows)

Reply:
I think it'll detect those things and reduces performance gracefully. I guess CI will tell us.